stylesheets/modules/mozugrid/grid.less

```diff
@@ -96,8 +96,8 @@
         border-bottom:none;
      }
 
-    .mz-grid-column {
-        padding: 0px 15px;
+    .mz-grid-column {        
```

Comment:
Not a good idea to override the styles of the generic grid component.  Make sure we are testing the old look and feel and not breaking anything in the existing grid. If you want you can override these styles only for your grid in specific CSS/LESS files.